src/server/handlers.rs

```diff
@@ -75,7 +78,18 @@ impl<P: Publisher> Handler<P> {
             }
         }
 
-        let manifest_name = manifest.metadata.name.clone();
+        let manifest_name = manifest.metadata.name.trim().to_string();
+        if manifest_name.contains(' ') || manifest_name.contains('.') {
```

Comment:
How about a simple regex for this?

Reply:
aw man don't make me regex

Reply:
In all seriousness, I'm fine doing that as long as whatever was going on with big regular expressions in our other Rust project isn't something that _normally_ happens.

Reply:
I _think_ that was because said regex was a bit more monstrous. How about this? `[-\w]+`